src/org/labkey/test/pages/ImportDataPage.java

```diff
@@ -118,6 +123,19 @@ public void cancel()
         clearCache();
     }
 
+    private File downloadTemplate()
+    {
+        return doAndWaitForDownload(()->elementCache().getDownloadTemplateButton().click());
+    }
+
+    public List<String> getTemplateColumnHeaders() throws IOException
+    {
+        File template = downloadTemplate();
+        Workbook workbook = ExcelHelper.create(template);
+        Sheet sheet = workbook.getSheetAt(0);
+        return ExcelHelper.getRowData(sheet, 0);
```

Comment:
I would have this function return just the file and put the ExcelHelper and Workbook code in the test case.

Reply:
done